cmd/broker.go

```diff
@@ -64,12 +69,20 @@ func init() {
 	brokerCmd.PersistentFlags().StringVarP(&brokerName, "name", "n", "", "Name of Automation Broker instance")
 	rootCmd.AddCommand(brokerCmd)
 
+	brokerCatalogCmd.Flags().StringVarP(&catalogOutputFormat, "output", "o", "", "Display broker catalog output in JSON or YaML format")
```

Comment:
Is it really YaML? The YAML site seems to always use YAML. http://yaml.org/

Reply:
Oh you're right.